### DISCUSSION TITLE
Include file contents in code block

### DISCUSSION THREAD

**Discussion by plessl** [quarto-dev/quarto-cli#2312] [Q&A] [answered]
Hi Quarto community.

I recently discovered Quarto and I'm very impressed by this wonderful software.

I'm currently preparing a companion site to a lecture using Quarto where it would be helpful to keep code examples in files and include the contents in an HTML code cell in order to avoid code duplication (and to work around some limitations in Quarto, Juypter and Julia).

I would like to write code like this:

````
```julia
{{< include example.jl >}}
```
````

Unfortunately, this doesn't work. Instead of the contents of `example.jl` Quarto treats `{{< include example.jl >}}` like a line of Julia code and prints it. Is this a bug? Or does the include need to be escaped somehow?

Thanks,
 Christian

**Comment by cderv** [comment, marked as answer]
I believe it should work as this was closed https://github.com/quarto-dev/quarto-cli/issues/1237

it seems broken again. I can reproduce on 1.1 and 1.2 and we are supposed to look into Code blocks: 
https://github.com/quarto-dev/quarto-cli/blob/0facbe6664e77be39b9a900cf9df29c8ffcb8b2a/src/resources/filters/quarto-pre/shortcodes.lua#L14

I'll create a new issue to track this.
